projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -65,9 +65,14 @@ function jetpack_add_dashboard_sub_nav_item() {
 	 * If user is allowed to see the Jetpack Admin, add Settings sub-link.
 	 *
 	 * @since 4.3.0
+	 * @since 9.7.0 If Connection does not have an owner, restrict it to admins
 	 */
 	function jetpack_add_settings_sub_nav_item() {
-		if ( ( ( new Status() )->is_offline_mode() || Jetpack::is_connection_ready() ) && current_user_can( 'edit_posts' ) ) {
+		if (
+			( ( new Status() )->is_offline_mode() || Jetpack::is_connection_ready() ) &&
+			current_user_can( 'edit_posts' ) &&
```

Comment:
Would it make more sense to check against the `jetpack_connect` capability instead, and bail early? I'm just not 100% sure that `edit_posts` will catch them all.

Reply:
+1 using `jetpack_connect` cap. See the latest changes and tell me that's what you thought